### PR TITLE
fix: Melange 3 revdeps

### PR DESCRIPTION
**Pull request by anmonteiro**

### CODE REVIEW

**Comment by arbipher** [comment]
I guess you can modify (force-pushed) your previous pr #25138 instead of making a new one. The CI should relaunch.

**Comment by mseri** [comment]
Thanks, I thin it is generally better to keep the revdeps separately to spare CI cycles

File: packages/tablecloth-melange/tablecloth-melange.0.0.9/opam [hunk 1]
```diff
@@ -16,7 +16,7 @@ depends: [
   "ocaml" {>= "5.1"}
   "dune" {>= "3.8"}
   "reason"
-  "melange"
+  "melange" {>= "2.0.0" & < "3.0.0"}
```
Comment (mseri) [inline packages/tablecloth-melange/tablecloth-melange.0.0.9/opam:19]: I assume the lower bound is intentional and was incorrect here, but I prefer always to double check :)

Comment (mseri) [reply packages/tablecloth-melange/tablecloth-melange.0.0.9/opam:19]: @anmonteiro please let me know and then I'll merge

Comment (anmonteiro) [reply packages/tablecloth-melange/tablecloth-melange.0.0.9/opam:19]: oops, not intentional. if this was tested with melange 1.0 then we can revert.

**Pull request merged**